README.md

```diff
@@ -39,6 +39,14 @@ Run this command to install Modelina in your project:
 npm install @asyncapi/modelina
 ```
 
+## AsyncAPI CLI
+
+Run this command to use [Modelina as part of the AsyncAPI CLI](https://github.com/asyncapi/cli#usage):
+
+```bash
+asyncapi generate models <language> ./asyncapi.json
+```
```

Comment:
You should add information here how to install AsyncAPI CLI - link to CLI repo to `Installation` section should be enough :)

Reply:
Makes sense 👍